code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -390,3 +424,44 @@
 	new /obj/item/paper/camera_bug(src)
 	for(var/i in 1 to 5)
 		new /obj/item/wall_bug(src, camera)
+
+/obj/item/storage/box/syndie_kit/bundle/badass/
```

Comment:
Not entirely sure on this being a box subtype. Maybe you can make this some sort of Beacon, like the Powersink uses or something?

Reply:
https://github.com/ParadiseSS13/Paradise/pull/23465/commits/5770ed791b394fd07e2eee10ff24603f40f104bb